### DISCUSSION TITLE
Template Suggestions

### DISCUSSION THREAD

**Discussion by shogunpurple** [Budibase/budibase#632] [Ideas]
As we start to build out template functionality, I'd love to hear some thoughts around:

- Which templates we should build
- Any that you may have created and would like to add to the platform

Some suggestions from previous discord chat:
- Sales CRM
- Todo List
- Contact List

**Comment by joebudi** [comment]
Hey. 

Additional popular use cases include:
- Inventory platform
- Membership software

I would heavily suggest an inventory platform as it really is a popular use case and will show off the new grid component.

**Reply by shogunpurple** [reply]
Yeah definitely seems like a common one - especially in logistics circles. The membership one can be implemented at a basic level but will be hard until we have more sophisticated access control

**Comment by shynes** [comment]
I think some ideas may be to complex to start, here are some smaller, more broad base ones. I would copy some ideas from the most popular themes in themeforest. But some smaller widgets could include

1. Blog type Content Pages/Posts
2. Landing pages/ email capture
3. Photo Album with Tags/Filters
4. Hero/Top Area - Dyanmic Content/ Pictures/ Slide Show

**Reply by ukane-philemon** [reply]
Love the idea of implementing a blog template.

**Reply by joebudi** [reply]
We've never considered Budibase as a platform for blogging to be completely honest. You could, in essence, create a platform like Substack though :-)

**Comment by thiagoaragao** [comment]
Accounts Payable/Receivable.

**Reply by joebudi** [reply]
Interesting. Could you elaborate on this a little? Are there any good tools out in the wild that would be similar to your desired template?

**Comment by Apexdestroyer13** [comment]
Project/File Index with geocoded map search

**Reply by joebudi** [reply]
Interesting. Is there a particular db you would like to use with such a template?

**Comment by yulin724** [comment]
IOT template with mainflux

**Reply by joebudi** [reply]
This is interesting. Thanks for the idea!

**Comment by fatihsever** [comment]
https://airtable.com/templates

All of the Airtable templates are amazing. Maybe you can be inspired by these templates.

**Reply by mjashanks** [reply]
For sure - such a great resource!

**Comment by PekkaJalonen** [comment]
Risk Management

**Reply by PekkaJalonen** [reply]
Could be bundled with IT templates, such as Asset Management and/or Project Approval template
